app/src/main/java/ch/epfl/sweng/swenggolf/offer/ListAnswerAdapter.java

```diff
@@ -43,6 +42,7 @@
     private static final int HEART_EMPTY = R.drawable.ic_favorite_border;
     private Answers answers;
     private Offer offer;
+    private boolean isClosed;
```

Comment:
I don't get exactly why you don't use the isClosed of offer.

Reply:
Because it can change during execution and I have not set a setter for this argument in offer. I could do it, but then I should use an observer pattern to notify the ListAnswerAdapter when the offer changes. I think it's too much complexity for one action. What do you think ?